gloo/docs/gloo_routing/virtual_services/security/custom_auth/_index.md

```diff
@@ -4,40 +4,77 @@ weight: 80
 description: External Authentication with your own auth server
 ---
 
-While Gloo provides an auth server that covers your OpenID Connect, ApiKey, and Basic Auth use cases, it
-also allows your to use your own authentication server, to implement custom auth logic.
+{{% notice note %}}
+The custom auth feature was introduced with **Gloo**, release 0.20.7, and **Gloo Enterprise**, release 0.13.5. 
+If you are using an earlier version, this tutorial will not work.
+{{% /notice %}}
 
-In this guide we will demonstrate your to create and configure Gloo to use your own auth service.
-For simplicity we will use an http service. Though this guide will work (with minor adjustments) also work with a gRPC server that implements
-the Envoy spec for an [external authorization server](https://github.com/envoyproxy/envoy/blob/master/api/envoy/service/auth/v2/external_auth.proto).
+Gloo Enterprise ships with an external auth server that implements a wide array of authentication and authorization models. 
+Even though these features are currently not available in the open source version of Gloo, you can deploy your own 
```

Comment:
> currently not available in the open source version of Gloo,

do we ever plan on open-sourcing this? this seems to imply we have plans to open source that in the future

Reply:
Not sure, that's why I added the adverb. Makes sense to remove it.